README.md

```diff
@@ -1,8 +1,53 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/java-apps/core_java)]:
+      
+  -  Twitter App: Currently working on the project.
+  -  JDBC App: Currently working on the project.
+  - Grep App: Implemented Java Grep application which is looking for a regular expression that is searched in each file. I wrote this program in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command line with the JRE installed or via a docker container.
+
+
+## Highlighted Projects
+**Airline reservation system** [[GitHub](https://github.com/Na-zi/Projects/tree/main/airline-reservation)]: Designed an Airline Reservation System application which enables the customers to search and book the flights. This project has been developed in Visual Studio using C# programming language and Microsoft Azure SQL Database is implemented to manage the database on Cloud. The unit test and integration test are done for the system. Managing software development is done by using GitHub. Planning and tracing Scrum processes and activities are done by Microsoft Azure DevOps. Flow diagram and context diagram are used to model different sprints.
+
+**Snake game** [[GitHub](https://github.com/Na-zi/Projects/tree/main/snake_game)]: Designed snake game using Python as Object Oriented Language. The snake game has a snake that you can control it by using your keyboard to grab the food. The game is developed in PyCharm. I broke the problem down into different seven steps to solve it.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**: Collaborated on a Scrum team within an Agile environment as a data engineer. Implemented software projects using a variety of technologies, such as Linux, Java, SQL, Docker, Git, and Google Cloud Platform, to familiarize with current industry trends.
+
+**Biomedical Engineer, 5-Azar hospital, Gorgan, Iran (2016-2018)**: Analyzed technical defects and implement data-driven solution with a team of engineers with engineering guidance. Prioritized workload effectively and complete all work assignments in a timely manner. Managed my team member hours to hit monthly targets while maintaining above 100% productivity are part of my roles.
```

Comment:
Analyzed technical defects and implemented data-driven solutions with a team of engineers with engineering guidance. Prioritized workload effectively and completed all work assignments in a timely manner. Managed my team member hours to hit monthly targets while maintaining above 100% productivity are part of my roles.